### ISSUE TITLE
The Hyper plan 2.0

### ISSUE THREAD

**Issue by albinekb** [zeit/hyper#2284]
This is an umbrella issue to track all things that need to be in place before 2.x can go `stable`

# Needed
- [x] https://github.com/zeit/hyper/issues/2233 Ability to map multiple keys per command
- [x] https://github.com/zeit/hyper/issues/2248 Can't change cursor
- [x] https://github.com/zeit/hyper/issues/2288 hyper(1) CLI
- [X] ~https://github.com/zeit/hyper/issues/2230 Close tab with middle click (regression)~ not a regression. Never worked
- [ ] https://github.com/zeit/hyper/issues/2297 Handle key repeat on macOS (regression)
- [x] [~Replace react@16rc3 with stable v16.0 release~](https://github.com/zeit/hyper/pull/2310)

# Nice to have (won't block release)
- [ ] New plugin API for terminal rendering (to support coming `xterm 3.0`)
- [x] Hyper Store

**Comment by LucaT1** [comment]
How could the community work on the Hyper store? I'd love to contribute to that piece of the software😍

**Comment by albinekb** [comment]
It will be open sourced when it's released 👍  @LucaT1

**Comment by Stanzilla** [comment]
Should probably fix all the resizing issues before it as well:
* https://github.com/zeit/hyper/issues/2252
* https://github.com/zeit/hyper/issues/1788
* https://github.com/zeit/hyper/issues/1657
* https://github.com/zeit/hyper/issues/476

**Comment by MAPESO** [comment]
@Stanzilla This problem has already been solved **--->** #2252 😄

**Comment by Stanzilla** [comment]
Removed it, thanks.

**Comment by Stanzilla** [comment]
https://github.com/zeit/hyper/issues/2633
https://github.com/zeit/hyper/issues/2592
https://github.com/zeit/hyper/issues/2239
https://github.com/zeit/hyper/issues/2574
https://github.com/zeit/hyper/issues/2548

**Comment by Stanzilla** [comment]
Also should list updating the docs as a bullet point, especially the extensions part and https://github.com/zeit/hyper/blob/canary/website/index.html#L712

**Comment by albinekb** [comment]
2.0 is out! 🔥
